cfr.json

```diff
@@ -1534,5 +1534,178 @@
         "www.netflix.com"
       ]
     }
+  },
+  {
+    "id": "CFR_EXP_FBW_PROMOTION",
```

Comment:
Just to confirm, is this an experiment? If not, let's remove the `EXP` in the `id`.